src/main/java/vazkii/botania/common/item/relic/ItemLokiRing.java

```diff
@@ -394,7 +383,6 @@ public static void breakOnAllCursors(EntityPlayer player, Item item, ItemStack s
 		World world = player.worldObj;
 		boolean silk = EnchantmentHelper.getEnchantmentLevel(Enchantment.silkTouch.effectId, stack) > 0;
 		int fortune = EnchantmentHelper.getEnchantmentLevel(Enchantment.fortune.effectId, stack);
-		boolean dispose = breaker == null? true : breaker.disposeOfTrashBlocks(stack);
```

Comment:
Is `breaker.disposeOfTrashBlocks(stack)` still needed?

Reply:
The current implementations (ItemTerraAxe and ItemTerraPick) more or less just return a boolean, so I would say no, we don't need it.